### PR TITLE
Layer5 welcome logo changed to Meshery logos with dark and light them…

### LINKED ISSUE #296: Replace the Logo of Layer5 with Meshery in config.yml workflow
**Description**

Currently, we are using the Logo of Layer5 instead of Meshery. This repository is under the organization Meshery, so, there should be an image of Meshery.

**How to resolve this**

- Go to .github and replace this welcome image with the new one:

- Make sure to display the images conditionally, based on the theme selected by user.
Here's the sample code:

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Meshery Logo" src="">
</picture>
```

New images are:

image

image

- Download these images, and upload with an appropriate name. Make sure to use conditional rendering as described above.

- After uploading the image, and removing the previous image, go to .github/config.yml folder and update the reference to previous image to the new image in the following part:

```
firstPRMergeComment: >
  Thanks for your contribution to the Layer5 and Meshery community! :tada:
   
  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
```
Here, replace `Congrats!` with the code

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
</picture>
```

make sure to change the image sources with the path to image in the folder where you upload it.

- That's it.

### PR DESCRIPTION
**Pull request by om2137**
**Description**
Replace the Logo of Layer5 with Meshery in the config.yml workflow

This PR fixes #296 also #295

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
```
Comment (thisiskaransgit) [inline .github/config.yml:18]:
```suggestion
  Thanks for your contribution to Meshery! :tada:
```

**Review by leecalcote — CHANGES_REQUESTED** [review]
@anshgoyalevil, please assist @om2137 by correcting the instructions offered in the corresponding issue.

File: .github/config.yml [hunk 2]
```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
...
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/layer5io/meshkit/master/.github/welcome/Layer5-celebration.png)
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./welcome/Meshery-celebration-w.png">
+    <source media="(prefers-color-scheme: light)" srcset="./welcome/Meshery-celebration-b.png">
+    <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
+  </picture>
```
Comment (anshgoyalevil) [inline .github/config.yml:24]: Thanks for the changes @om2137 🛩️ 

There are some changes required.

- The ```srcset``` needs to be a reference to absolute path to raw image.
- The name of the images needs to be more clear, as explained by @thisiskaransgit 
- Also, I think, you have interchanged the white image to dark color scheme, and black to light color scheme.

Please refer to the following example

```
Thanks for your contribution to Meshery! :tada:
   
  <picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Meshery-celebration-Dark.svg">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Meshery-celebration-Light.svg">
  <img alt="Meshery Logo" src="">
  </picture>
```

Or, Following Merged PR, referring to a similar issue - https://github.com/meshery/meshery/pull/7586/files

**Comment by goldfishdolphin** [comment]
@om2137 are you still working on the issue?

**Comment by anshgoyalevil** [comment]
@om2137 Are you still working on the issue? Feel free to ping me here or on slack if you are facing any issues.

**Comment by om2137** [comment]
> @om2137 Are you still working on the issue? Feel free to ping me here or on slack if you are facing any issues.

Hey extremely sorry, I didn't get the notification for this.
but I thought some one already fixed this issue in other pr.

If you want you can take this

**Comment by anshgoyalevil** [comment]
@om2137 Alright 😅 Would you like to close this PR?

**Pull request merged**